providers/catalog.go

```diff
@@ -1972,4 +1973,29 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Teamwork configuration
+	Teamwork: {
+		AuthType: Oauth2,
+		BaseURL:  "https://{{.workspace}}.teamwork.com",
```

Comment:
Is it possible to get the `workspace` value from their API? If we can, we can set the `PostAuthInfo` to true, and add a `AuthMetadataConnector` for this provider (check https://github.com/amp-labs/connectors/pull/319 as an example)